mongo/cursor.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 // Cursor is used to iterate over a stream of documents. Each document can be decoded into a Go type via the Decode
-// method or accessed as raw BSON via the Current field.
+// method or accessed as raw BSON via the Current field. This type is not goroutine safe and must not be used
+// concurrently by multiple goroutines.
```

Comment:
Nice, I see this warning on the `SessionContext` type too.

DRIVERS-1789 also notes change stream cursors, since a user may be blocked on the change stream's `Next` call indefinitely. To unblock, users can cancel the passed context. But I think the motivation for the documentation is to make it clear that you cannot call `Close` in a separate goroutine in that case.

Can you add the same note to the `ChangeStream` type?

Reply:
done